policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl

```diff
@@ -1,7 +1,7 @@
 {{ define "goreleaser" }}
   goreleaser:
     name: '{{`${{ matrix.golang_cross }}`}}'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-m-2
```

Comment:
We have two larger runners. I think we can do builds on one and tests on the other. ubuntu-latest-m-2 is the larger one. I forgot the name of the other one, @buger could you take a look at the Github org settings and tell us?

Reply:
This ones? 
<img width="950" alt="image" src="https://github.com/TykTechnologies/gromit/assets/14009/e54132e5-a3be-43c7-a5ab-6bf68231a7a1">

Reply:
Yes, thank you.

Reply:
I will check the performance on m-2 and m machines and post the results